src/wagtail_personalisation/migrations/0022_personalisablepagemetadata_canonical_protect.py

```diff
@@ -7,7 +7,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('wagtail_personalisation', '0020_rules_delete_relatedqueryname'),
+        ('wagtail_personalisation', '0021_personalisablepagemetadata_segment_set_on_delete_protect'),
```

Comment:
https://github.com/wagtail/wagtail-personalisation/blob/e42e1a865be842be64734a211044c70485dff748/src/wagtail_personalisation/migrations/0021_personalisablepagemetadata_canonical_protect.py#L19-L22

This also modifies the variant field, so may be worth regenerating the migration or not altering the variant field in the migration at all.

Reply:
Thank you! I've split the migrations 👍